controllers/auth/updateAvatar.js

```diff
@@ -0,0 +1,30 @@
+const path = require("path");
+const fs = require("fs/promises");
+const Jimp = require("jimp");
+
+const { User } = require("../../models");
+
+const avatarDir = path.join(__dirname, "../../", "public", "avatars");
+
+const updateAvatar = async (req, res) => {
+  const { _id } = req.user;
+  const { path: tempUpload, originalname } = req.file;
```

Comment:
originalname should be originalName. In JS we use camelCased declared variables, even for destructured ones.